packages/frontend/src/apollo/types.ts

```diff
@@ -1,9 +1,18 @@
-import { AddFlockInputDTO, FlockDTO, UserDTO } from '@flocker/api-types';
+import {
+  AddFlockInputDTO,
+  FlockAvailabilityDTO,
+  FlockDTO,
+  UserAvailabilityIntervalDTO,
+  UserDTO,
+} from '@flocker/api-types';
 
 // Results
 export type GetCurrentUserResult = { getCurrentUser: UserDTO };
 export type AddFlockResult = { addFlock: FlockDTO };
 export type JoinFlockResult = { joinFlock: { flockCode: string } };
+export type GetCurrentFlockResult = { getFlockByCode: FlockDTO };
+export type GetUserIntervalsResult = { getUserIntervals: UserAvailabilityIntervalDTO };
+export type GetFlockIntervalsResult = { getUserIntervalsForFlock: FlockAvailabilityDTO };
```

Comment:
Should there be input types as well?

Reply:
For user intervals yes (when we can manually select) but not for flock intervals righttt